components/dashboards-web-component/package-lock.json

```diff
@@ -3590,6 +3590,16 @@
         "jsx-ast-utils": "1.4.1"
       }
     },
+    "eslint-plugin-prettier": {
```

Comment:
Let's not commit package-lock.json file as you have not added any new node modules.